composer.json

```diff
@@ -7,7 +7,7 @@
     },
     "autoload": {
         "psr-4": {
-            "Ripcord\\": "src/Ripcord"
+            "Ripcord\\": "src/Ripcord/"
```

Comment:
We should not need `/` at the end. Why this was added?

Reply:
Commit #ea4f923270b3c061bdc8412755173c042aee3eaf